pixeldata/src/lib.rs

```diff
@@ -1522,6 +1572,14 @@ impl DecodedPixelData<'_> {
     /// }
     /// ```
     pub fn to_owned(&self) -> DecodedPixelData<'static> {
+        let voi_lut_sequence = self
+            .voi_lut_sequence
+            .as_ref()
+            .map(|voi_lut| VoiLutSequence {
+                signed: voi_lut.signed,
+                first_mapped_value: voi_lut.first_mapped_value,
+                lut_data: voi_lut.lut_data.clone(),
+            });
```

Comment:
I'm prob just missing something obvious, but how is mapping here different from clone?